torch/csrc/jit/frontend/script_type_parser.cpp

```diff
@@ -243,6 +238,28 @@ TypePtr ScriptTypeParser::parseTypeFromExprImpl(const Expr& expr) const {
       return custom_class_type;
     }
 
+    // `torch.cuda.Stream` and `torch.cuda.Event` are aliased as
+    // custom classes of type torch.classes.cuda.Stream and
+    // torch.classes.cuda.Event respectively. Return the respective
+    // custom class types for these two cases.
+    if (type_name.find("torch.cuda.Stream") == 0) {
+      auto custom_class_type =
+          getCustomClass("__torch__.torch.classes.cuda.Stream");
+      return custom_class_type;
+    }
+
+    if (type_name.find("torch.cuda.Event") == 0) {
+      auto custom_class_type =
+          getCustomClass("__torch__.torch.classes.cuda.Event");
+      return custom_class_type;
+    }
+
+    if (resolver_) {
+      if (auto typePtr = resolver_->resolveType(type_name, expr.range())) {
+        return typePtr;
+      }
+    }
```

Comment:
I hope moving this doesn't break any tests...

Reply:
All the tests have passed. The failures seen below are due to issues in reporting the results.